framework/db/mysql/QueryBuilder.php

```diff
@@ -217,4 +217,112 @@ public function insert($table, $columns, &$params)
+        );
+    }
+
+    /**
+     * Builds a SQL command for adding comment to table
+     *
+     * @param string $table the table whose column is to be commented. The table name will be properly quoted by the method.
+     * @param string $comment the text of the comment to be added. The comment will be properly quoted by the method.
+     * @return $this the command object itself
+     */
+    public function addCommentOnTable($table, $comment)
+    {
+        $quotedTable = $this->db->quoteTableName($table);
+        return sprintf(
+            'ALTER TABLE %s COMMENT %s',
+            $quotedTable,
+            $this->db->quoteValue($comment)
+        );
+    }
+
+    /**
+     * Builds a SQL command for adding comment to column
+     *
+     * @param string $table the table whose column is to be commented. The table name will be properly quoted by the method.
+     * @param string $column the name of the column to be commented. The column name will be properly quoted by the method.
+     * @return $this the command object itself
+     */
+    public function dropCommentFromColumn($table, $column)
+    {
+        $quotedTable = $this->db->quoteTableName($table);
+        $definition = $this->getColumnDefinition($quotedTable, $column);
+        $definition = trim(preg_replace("/COMMENT '(.*?)'/i", '', $definition));
+        return sprintf(
+            'ALTER TABLE %s CHANGE %s %s%s COMMENT \'\'',
+            $quotedTable,
+            $this->db->quoteColumnName($column),
+            $this->db->quoteColumnName($column),
+            empty($definition) ? '' : ' ' . $definition
+        );
+    }
+
+    /**
+     * Builds a SQL command for adding comment to table
+     *
+     * @param string $table the table whose column is to be commented. The table name will be properly quoted by the method.
+     * @return $this the command object itself
+     */
+    public function dropCommentFromTable($table)
+    {
+        $quotedTable = $this->db->quoteTableName($table);
+        return sprintf(
+            "ALTER TABLE %s COMMENT ''",
+            $quotedTable,
+            empty($definition) ? '' : ' ' . $definition
+        );
+    }
+
+
+    /**
+     * Gets column definition.
+     *
+     * @param string $table table name
+     * @param string $column column name
+     * @return null|string the column definition
+     * @throws Exception in case when table does not contain column
+     */
+    private function getColumnDefinition($table, $column)
+    {
+        $row = $this->db->createCommand('SHOW CREATE TABLE ' . $table)->queryOne();
+        if ($row === false) {
+            throw new Exception("Unable to find column '$column' in table '$table'.");
+        }
+        if (isset($row['Create Table'])) {
+            $sql = $row['Create Table'];
+        } else {
+            $row = array_values($row);
+            $sql = $row[1];
+        }
+        if (preg_match_all('/^\s*`(.*?)`\s+(.*?),?$/m', $sql, $matches)) {
```

Comment:
Unfortunaly you can not use the same way for getting the column definition like for mysql. Here is an result of `show create table` in cubdir 9.3

```
1'th query[ 0,14 second , count of total rows : 1 ]
--------------------------------------------------
SHOW CREATE TABLE animal;
--------------------------------------------------
animal	CREATE TABLE [animal] ([id] INTEGER AUTO_INCREMENT (1, 1) NOT NULL, [type] CHARACTER VARYING(255) NOT NULL,  CONSTRAINT [pk_animal_id] PRIMARY KEY  ([id])) COLLATE utf8_bin
```

You can find how i tried to implement this feature https://github.com/unclead/yii2/blob/8152-extend-migration-class/framework/db/cubrid/QueryBuilder.php#L64 but you can try to write more beautifull regex